destination.go

```diff
@@ -1,75 +1,79 @@
-package connectorname
+package pinecone
 
 //go:generate paramgen -output=paramgen_dest.go DestinationConfig
 
 import (
 	"context"
 	"fmt"
-
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
 type Destination struct {
 	sdk.UnimplementedDestination
 
 	config DestinationConfig
+	writer *Writer
 }
 
 type DestinationConfig struct {
-	// Config includes parameters that are the same in the source and destination.
-	Config
-	// DestinationConfigParam must be either yes or no (defaults to yes).
-	DestinationConfigParam string `validate:"inclusion=yes|no" default:"yes"`
+	// PineconeAPIKey is the API Key for authenticating with Pinecone.
+	PineconeAPIKey string `json:"pinecone.apiKey" validate:"required"`
+
+	// PineconeHostURL is the Pinecone index host URL
+	PineconeHostURL string `json:"pinecone.hostURL" validate:"required"`
```

Comment:
I believe it's enough that we call the params simply `APIKey` and `HostURL`, i.e. without the `pinecone` prefix. It's all within the connector, so it's clear that it's about Pinecone.

Reply:
Solved in a later pr